### ISSUE TITLE
Android get a rejection on configure with error code : 2

### ISSUE THREAD

**Issue by avencat** [joonhocho/react-native-google-sign-in#11]
Hi, first of all, thank you for the great work !

I open this issue because I'm having an error 2 rejection when calling configure without any error message so I don't know what's wrong in my configuration. :/

I'm using RN 0.39 and I'm calling configure with the following arguments:

```javascript
await GoogleSignIn.configure({
clientID: Platform.os === 'ios' ? '[myKey].apps.googleusercontent.com' : null,
shouldFetchBasicProfile: true,
serverClientID: '[myServerKey].apps.googleusercontent.com',
offlineAccess: true,
}).catch(err => {
    console.log('err', err);
});
```

And the console.log gets:
```json
{
    code: 2
}
```

This code works just fine on iOS but not in Android and I don't know what I'm doing wrong... :/

If somebody knows what error code 2 is, can you help me please ?

Thank you in advance and once again, great work !

**Comment by avencat** [comment]
I ran into this issue with your ExampleApp also, I think there might be an error on my system but I'd like to know what this error 2 means... :/

EDIT: This is really a strange behavior because when I catch the error from configure, both in your example and in my project, I'm able to connect to Google and retrieve the user informations but with a google blank page (no logo or name of my application)

**Comment by joonhocho** [comment]
@avencat Can you make sure that your google play services is up to date on your Android device? Also, make sure you have followed the instructions from Google's official guide setting up your application with Google and provided correct configurations.

**Comment by avencat** [comment]
Well, I tried it on a fresh android studio install, with an emulator running Android 7.1.1 that should include google play services but I will try it on a real device tomorrow and I will keep you informed ! 😉

**Comment by avencat** [comment]
I just tested on a real device and it works just well ! Thanks a lot ! (If you know how to install Google Play Services on an Android Studio emulator, it would be great ! :D)

Thanks again for your work ! :)

**Comment by joonhocho** [comment]
@avencat Awesome.

**Comment by mvdam** [comment]
This issue is closed but it is still a problem on the Android emulator. It keeps giving `code 2` errors. On (emulated) iOS it works fine. For Android only on actual devices it's working.

I've tried different emulator setups; Android 6.0, 7.1.1. API 23, 25. All with Google API's enabled.

Could you please look into it again?

**Comment by mvdam** [comment]
I found out that an update of Google Play Services fixed it! Now running v11.0.55 and logging in on the emulator works! Thanks for your great work @joonhocho!

**Comment by trinadhkoya** [comment]
I'd like to know what this error 2 means... :/
